frontend/src/views/LevelView.vue

```diff
@@ -0,0 +1,156 @@
+<template>
+  <div class="Container">
+    <img src="@/assets/images/logo-and-lettering.svg" alt="Saci Logo" />
+    <h1>{{ newLevelTitle }}</h1>
+    <h2>Basic Information</h2>
+    <div class="NameContainer">
+      <h5>Name</h5>
+      <ElInput placeholder="Level Name" v-model="newLevel.name" />
+    </div>
+    <div class="FlexContainer">
+      <div class="MaxAndMinInputs">
```

Comment:
Do you need this div?

An almost 10% rule: if you have a div, and you have just one div inside the first one, the second one is (almost sure) unnecessary